### PR TITLE
Updated Custom Domains Docs for Launch

### PR DESCRIPTION
**Pull request by r-birkner**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/production/custom-domain.md [hunk 1]
```diff
@@ -1,27 +1,170 @@
...
 
-Ultimately the security of any site using standard web technology depends on DNS since control of DNS allows the site to be redirected and enables control of TLS certificates. Consequently, for a standard website trust must be placed at least in the DNS registrar. If the registrar provides static hosting, deployment of a custom service worker can provide end-to-end security for standard Web3 dapps on the Internet Computer blockchain without increasing the number of entities that must be trusted.
+1. Register the domain with the boundary nodes;
+1. Hosting the domain on your own infrastructure.
 
-## Creating the custom Service Worker
+For both approaches, you need to acquire a domain through your favorite registrar.
+
+The two approaches differ in the ease of use and the configurability. When registering
+the domain with the boundary nodes, you simply has to configure the DNS records of
```
Comment (sesi200) [inline docs/developer-docs/production/custom-domain.md:15]:
```suggestion
the domain with the boundary nodes, you simply have to configure the DNS records of
```

File: docs/developer-docs/production/custom-domain.md [hunk 2]
```diff
@@ -1,27 +1,170 @@
...
+the custom domain and the boundary nodes take care of obtaining a certificate,
+renewing the certificate before its expiration, SEO and serving the service worker.
+When hosting the domain on your own infrastructure, you need to obtain and renew
+the certificates, and to provide your own infrastructure that serves the service worker.
```
Comment (sesi200) [inline docs/developer-docs/production/custom-domain.md:19]:
```suggestion
the certificates, and provide your own infrastructure that serves the service worker.
```

File: docs/developer-docs/production/custom-domain.md [hunk 3]
```diff
@@ -1,27 +1,170 @@
...
+a concrete example. Finally, we explain how one can update and remove a registration.
+
+### First Registration
+
+1. Configure the DNS record of your domain, which we denote with `CUSTOM_DOMAIN`.
+    * Add a `CNAME` entry for your domain pointing to `ic0.app` such that all the traffic destined to your domain is redirected to the boundary nodes;
+    * Add a `TXT` entry containing the canister ID to the `_canister-id`-subdomain of your domain (e.g., `_canister-id.CUSTOM_DOMAIN`);
+    * Add a `CNAME` entry for the `_acme-challenge`-subdomain (e.g., `_acme-challenge.CUSTOM_DOMAIN`) pointing to `_acme-challenge.CUSTOM_DOMAIN.icp2.io` in order for the boundary nodes to acquire the certificate.
+1. Create a file named `ic-domains` in your canister under `.well-known` containing the custom domain.
+    * By default, `dfx` excludes all files and directories, whose names starts with a `.`, from the asset canister. Hence, to include the `ic-domains`-file, you need to create an additional file, called `.ic-assets.json`.
```
Comment (sesi200) [inline docs/developer-docs/production/custom-domain.md:36]:
```suggestion
    * By default, `dfx` excludes all files and directories whose names start with a `.` from the asset canister. Hence, to include the `ic-domains`-file, you need to create an additional file, called `.ic-assets.json`.
```

**Review by sesi200 — COMMENTED** [review]
Can you also make it so that
1. the custom domain thing is not duplicated anymore and
2. the folder has a nice name?

You can control that stuff with the file `_category_.yml`. Have a look in other files of that name for examples.
image

**Review by harpa12 — APPROVED** [review]

**Review by csepreghy — APPROVED** [review]

**Pull request merged**